config/DECam/calibrate.py

```diff
@@ -47,9 +47,6 @@
         matchConfig.matcher.maxMatchDistArcSec = 2.0
         matchConfig.sourceSelector.active.excludePixelFlags = False
 
-# Demand astrometry and photoCal succeed
-config.requireAstrometry = True
-config.requirePhotoCal = True
```

Comment:
I do not understand the logic behind this commit. **We**, specifically, need astrometry and photometry, regardless of what the task default is or how any upstream pipelines are configured.